cmd/vc-rest/startcmd/start_test.go

```diff
@@ -215,6 +196,41 @@ func TestCreateKMS(t *testing.T) {
 	})
 }
 
+func setEnvVars(t *testing.T) {
+	err := os.Setenv(hostURLEnvKey, "localhost:8080")
```

Comment:
also backup the original env value and reset it in unsetEnvVars() if it exists

Reply:
Talked to @Baha-sk, the env vars seem to get wiped after tests anyway, so this isn't needed